config/backup.php

```diff
@@ -102,6 +102,11 @@
          */
         'database_dump_file_timestamp_format' => null,
 
+        /*
+         * If specified, the database dumped file name will contain the connection name in place of the database name.
+         */
+        'database_dump_file_use_connection_name' => null,
```

Comment:
since this is a bool value, shoud it be `false` by default instead of `null`?

Reply:
Yeah, I agree,  `false` would be a better default

Reply:
I agree. Had a thought though to change the config to a string, either 'database' or 'connection' and default it to connection.

I updated to that. It feels better that the 'database' option is visible in the config as default instead of a boolean to change behavior.

I'm happy to change it back to a boolean if that is preferred.